calyx/src/ir/printer.rs

```diff
@@ -522,4 +522,13 @@ impl Printer {
             ),
         }
     }
+
+    /// Formats the top-level metadata if present
+    pub fn format_metadata(metadata: &Option<String>) -> String {
```

Comment:
Can you add a call to this in the `main.rs` and add a new test for metadata stuff in `tests/parsing`?

Reply:
Nvm, I just didn't see it!